application/F3DStarter.cxx

```diff
@@ -320,10 +321,14 @@ int F3DStarter::Start(int argc, char** argv)
       }
     }
 
+    auto forceNoFileRenderVar =
+      F3DSystemTools::GetVectorEnvironnementVariable("CTEST_F3D_NO_FILE_RENDER_TESTING");
```

Comment:
What do you get a vector instead of a simple string?  
Moreover, do we care about the content? I guess just checking if the variable is defined is enough?

Reply:
fair enough